.gitignore

```diff
@@ -25,7 +25,6 @@ SpeexDSP.spec
 config.h
 config.log
 config.status
-include/speex/speexdsp_config_types.h
```

Comment:
Current hack in place.  config.h generates properly on osx cli.  Does not generate in android build.